tests/unit/flow.test.js

```diff
@@ -232,5 +232,45 @@ describe('Flow', () => {
         expect(result).toBe('repair-kitchen-problems');
       });
     });
+
+    describe('when a repairProblemBestDescription is selected', () => {
+      describe('when the repairProblemBestDescription is changed', () => {
+        test('then a repairProblemBestDescription is deselected and a repairProblemBestDescription does not exists', () =>{
```

Comment:
Both the tests have the same description and I believe this one needs to be corrected as the expect has `repairProblemBestDescription`